sdk/core/core-http/src/pipelineOptions.ts

```diff
@@ -62,4 +62,9 @@ export interface InternalPipelineOptions extends PipelineOptions {
    * Options to configure request/response logging.
    */
   loggingOptions?: LogPolicyOptions;
+
+  /**
+   * Configure whether to decompress response according to Accept-Encoding header (node-fetch only)
+   */
+  decompressResponse?: boolean;
```

Comment:
Are InternalPipelineOptions exposed to customers?

Reply:
I think it's exposed in public api surface, but with Internal in the name to discourage customers to use.

Reply:
They are not meant to be used by customers and we don't accept this interface in public API signatures. Obviously that doesn't stop someone from using the parameters, but what can you do :)

Reply:
Oh, is this something we should be exposing then? (I guess it's safe not to expose it and we can always expose it later if asked)

Reply:
but it IS exposed. I've seen usages, e.g., https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/textanalytics/ai-text-analytics/src/textAnalyticsClient.ts#L8